app/models/intake/gyr_intake.rb

```diff
@@ -349,7 +351,11 @@ class Intake::GyrIntake < Intake
   enum widowed: { unfilled: 0, yes: 1, no: 2 }, _prefix: :widowed
   enum wants_to_itemize: { unfilled: 0, yes: 1, no: 2, unsure: 3 }, _prefix: :wants_to_itemize
   enum received_advance_ctc_payment: { unfilled: 0, yes: 1, no: 2, unsure: 3 }, _prefix: :received_advance_ctc_payment
-
+  scope :accessible_intakes, -> do
```

Comment:
I feel like it would be a good idea to write some test for these scope methods -- there werent test coverage in them in the client logins file, but probably we should write a couple.